__tests__/app.test.js

```diff
@@ -78,6 +78,155 @@ describe("/api/reviews", () => {
           expect(body.msg).toBe("Route not found");
         });
     });
+
+
+    describe("queries", () => {
+      test("accepts category query", () => {
+        return request(app)
+          .get("/api/reviews?category=dexterity")
+          .expect(200)
+          .then(({ body }) => {
+            expect(body.reviews).toBeInstanceOf(Array);
+            expect(body.reviews).toHaveLength(1);
+
+            expect(body.reviews[0]).toMatchObject({
+              category: "dexterity",
+              comment_count: 3,
+              created_at: "2021-01-18T10:01:41.251Z",
+              designer: "Leslie Scott",
+              owner: "philippaclaire9",
+              review_id: 2,
+              review_img_url:
+                "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+              title: "Jenga",
+              votes: 5,
+            });
+
+            expect(body.reviews).toBeSortedBy("created_at", {
+              descending: true,
+            });
+          });
+          
+      });
+
+      test("accepts sort_by query", () => {
+        return request(app)
+          .get("/api/reviews?sort_by=votes")
+          .expect(200)
+          .then(({ body }) => {
+            expect(body.reviews).toBeInstanceOf(Array);
+            expect(body.reviews).toHaveLength(13);
+
+            body.reviews.forEach((review) => {
```

Comment:
No real need to have the assertion for the shape of all objects here - you've done that thoroughly above - you could just have the assertion for sort order